test/src/main/java/org/apache/accumulo/test/functional/RowDeleteIT.java

```diff
@@ -105,8 +106,7 @@ public void run() throws Exception {
       }
 
       try (Scanner scanner = c.createScanner(tableName, Authorizations.EMPTY)) {
-        count = Iterators.size(scanner.iterator());
-        assertEquals(0, count, "count == " + count);
+        assertTrue(scanner.stream().findAny().isEmpty());
         bw.close();
```

Comment:
Your change looks fine, but it looks like there's something weird with the existing `bw.close()`. It won't be closed if the assertion fails. The BatchWriter should be in a try-with-resources block of its own, wrapping this scanner block and other stuff above here. That way, it's always closed.